service/frontend/interface.go

```diff
@@ -22,7 +22,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-//go:generate mockgen -copyright_file ../../LICENSE -package $GOPACKAGE -source $GOFILE -destination interface_mock.go
+//go:generate mockgen -copyright_file ../../LICENSE -package workflowservicemock -destination ../../api/workflowservicemock/v1/service_grpc.pb.mock.go go.temporal.io/api/workflowservice/v1 WorkflowServiceClient
+//go:generate mockgen -copyright_file ../../LICENSE -package operatorservicemock -destination ../../api/operatorservicemock/v1/service_grpc.pb.mock.go go.temporal.io/api/operatorservice/v1 OperatorServiceClient
```

Comment:
Generating workflow & operator service mocks here.

Reply:
👍 too bad we made these part of `api-go` when they can just be generated by the library's user.